packages/manager/modules/telecom-spare/src/modems/order/modems-order.html

```diff
@@ -0,0 +1,236 @@
+<div class="telecom-spare-modems-order-section">
+    <header>
+        <tuc-section-back-link data-tuc-section-back-link-to-state="spare.modems"
+                            data-tuc-section-back-link-title="{{ ::'modems_order_back_link' | translate }}">
+        </tuc-section-back-link>
+        <h1 data-translate="modems_order_new_modem_title"></h1>
+    </header>
+
+    <div class="modems-order-section-content">
+
+        <tuc-toast-message></tuc-toast-message>
+
+        <div class="row">
+            <div class="col">
+                <!-- ORDER -->
+                <section class="widget-presentation"
+                         data-ng-if="$ctrl.order.orderURL">
+                    <h2 class="widget-presentation-title" data-translate="modems_order_shipping_title"></h2>
+                    <div class="alert alert-success" role="alert" data-translate="modem_order_order_success"></div>
+                    <p>
+                        <a target="_blank" class="oui-button oui-button_primary" rel="noopener"
+                            data-ng-href="{{:: $ctrl.order.orderURL }}">
+                            <span data-translate="modem_order_order_url"></span>
+                            <i class="fa fa-external-link" aria-hidden="true"></i>
```

Comment:
```suggestion
                            <span class="oui-icon oui-icon_external-link" aria-hidden="true"></i>
```
please try to use `oui` icons as much as possible

Reply:
Sorry, but it seems the icon isn't been displayed correctly with your suggestion

Reply:
My bad it's `.oui-icon-external_link`
 https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/icons